presto-hive/src/main/java/com/facebook/presto/hive/HdfsConfigurationUpdater.java

```diff
@@ -107,6 +109,8 @@ public void updateConfiguration(Configuration config)
     {
         copy(resourcesConfiguration, config);
 
+        config.setInt(LineRecordReader.MAX_LINE_LENGTH, toIntExact(new DataSize(100, DataSize.Unit.MEGABYTE).toBytes()));
```

Comment:
Why not something closer to the original limit, which was 20x bigger?

Reply:
So more thinking, I think we should handle this like we do the `dfsTimeout` config option in this method... there is a value that comes from the `HiveClientConfig`, and we always apply that.  
We would still need a good default value

Reply:
Once this becomes configurable, I am not that much concerned about the value.. The only problem would be that in case of a too long line, exception message would not be helpful in tracing the config switch governing the behavior. 
As this is supposed to be OOM-preventing safety mechanism only, we can start with 100M being the default and revisit the decision if we face any problems with this.